build/sign.proj

```diff
@@ -9,57 +9,75 @@
       BuildRTM=$(BuildRTM);
     </CommonMSBuildProperties>
     <OutDir>$(RepositoryRootDirectory)</OutDir>
-    <SignTargetsDependOn>BatchSign</SignTargetsDependOn>
-    <SignTargetsDependOn Condition="'$(SignPackages)' == 'true'">GetOutputNupkgs</SignTargetsDependOn>
-    <SignTargetsDependOn Condition="'$(SignPackages)' == 'true' AND '$(BuildRTM)' != 'true'">GetOutputNupkgs;GetOutputVsix</SignTargetsDependOn>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
-  
+
   <ItemGroup>
-    <FilesToSign Include="$(ArtifactsDirectory)microsoft.web.xdt\lib\net40\**\Microsoft.Web.XmlTransform.resources.dll">
-      <Authenticode>Microsoft400</Authenticode>
-      <StrongName>67</StrongName>
-    </FilesToSign>
+    <PackageReference Include="Microsoft.DotNet.SignTool" PrivateAssets="all" Condition="'$(UsingToolXliff)' == 'true'" />
```

Comment:
Can we add these PackageReference as GlobalPackageReference in the Directory.Packages.props file?